issue.go

```diff
@@ -53,7 +53,7 @@ type ChangelogItems struct {
 type ChangelogHistory struct {
 	Id      string           `json:"id" structs:"id"`
 	Author  User             `json:"author" structs:"author"`
-	Created string           `json:"created" structs:"created"`
+	Created Time           `json:"created" structs:"created"`
```

Comment:
This will break all code that is expecting a string.  Also, there are no tests to check that this actually works in different situations (bad format, empty string, positive case, etc).

If you want to add a field or possibly just add a function that returns the field as a `Time`.  Something like `CreatedAsTime()`, that would work.